client/my-sites/email/email-forwarding/email-forwarding-add-new-compact-list.jsx

```diff
@@ -0,0 +1,167 @@
+import { Button } from '@automattic/components';
+import { localize } from 'i18n-calypso';
+import PropTypes from 'prop-types';
+import { Component, Fragment } from 'react';
+import { connect } from 'react-redux';
+import { validateAllFields } from 'calypso/lib/domains/email-forwarding';
+import EmailForwardingAddNewCompact from 'calypso/my-sites/email/email-forwarding/email-forwarding-add-new-compact';
+import { composeAnalytics, withAnalytics } from 'calypso/state/analytics/actions';
+import { addEmailForward } from 'calypso/state/email-forwarding/actions';
+import getEmailForwardingLimit from 'calypso/state/selectors/get-email-forwarding-limit';
+import {
+	addEmailForwardSuccess,
+	getEmailForwards,
+	isAddingEmailForward,
+} from 'calypso/state/selectors/get-email-forwards';
+import { isRequestingSiteDomains } from 'calypso/state/sites/domains/selectors';
+import { getSiteSlug } from 'calypso/state/sites/selectors';
+import { getSelectedSiteId } from 'calypso/state/ui/selectors';
+
+class EmailForwardingAddNewCompactList extends Component {
+	static propTypes = {
+		emailForwards: PropTypes.array,
+		emailForwardingLimit: PropTypes.number,
+		onConfirmEmailForwarding: PropTypes.func.isRequired,
+		onAddForwardSuccess: PropTypes.func,
```

Comment:
Same remark about naming:

```suggestion
		onAddEmailForwardSuccess: PropTypes.func,
```